commit 7f7f06a15c11ade429ddeac346c0b4288f690ac3
Author: Free Ekanayaka <free.ekanayaka@canonical.com>
Date:   Sat Apr 14 14:29:00 2018 +0000

    Pass a logger to raft-http
    
    This will use the raftLogger wrapper that we already have, and forward to our
    logger. The output will be just a few INFO messages when connecting between
    nodes, joining and leaving, so pretty low noise.
    
    Signed-off-by: Free Ekanayaka <free.ekanayaka@canonical.com>

diff --git a/lxd/cluster/raft.go b/lxd/cluster/raft.go
index 97483553..72ce4c43 100644
--- a/lxd/cluster/raft.go
+++ b/lxd/cluster/raft.go
@@ -308,7 +308,7 @@ func raftNetworkTransport(
 	logger *log.Logger,
 	timeout time.Duration,
 	dial rafthttp.Dial) (raft.Transport, *rafthttp.Handler, *rafthttp.Layer, error) {
-	handler := rafthttp.NewHandler()
+	handler := rafthttp.NewHandlerWithLogger(logger)
 	addr, err := net.ResolveTCPAddr("tcp", address)
 	if err != nil {
 		return nil, nil, nil, errors.Wrap(err, "invalid node address")
